modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetricsProcessListView.macro

```diff
@@ -0,0 +1,19 @@
+definition {
+
+	macro assertPaginationComponentPresent {
+		AssertElementPresent(locator1 = "WorkflowMetricsProcessListView#PAGINATION_PREVIOUS_BUTTON");
+
+		AssertElementPresent(
+			key_pageNumber = "${pageNumber1}",
+			locator1 = "WorkflowMetricsProcessListView#PAGINATION_NUMBER_BUTTON");
+
+		if (isSet(pageNumber2)) {
+			AssertElementPresent(
+				key_pageNumber = "${pageNumber2}",
+				locator1 = "WorkflowMetricsProcessListView#PAGINATION_NUMBER_BUTTON");
+		}
+
+		AssertElementPresent(locator1 = "WorkflowMetricsProcessListView#PAGINATION_NEXT_BUTTON");
```

Comment:
You don't need this to assert that the Pagination Component is present. Please, create a path with the following locator:
`//div[contains(@class,'pagination-bar')]`

Then, just assert for it. Like this:
`AssertElementPresent(locator1 = "WorkflowMetricsProcessListView#PAGINATION_BAR");`

Reply:
okay, so i delete these paths now? since i'm not using it anymore :)

Reply:
Yes, you can delete it.